### DISCUSSION TITLE
how to change cursor shape on nvim with lua language ?

### DISCUSSION THREAD

**Discussion by iafshinafshin** [neovim/neovim#26762] [General]
### Problem

how to change cursor shape on neovim with lua language ?

look like this :
![Screenshot from 2023-12-27 20-16-40](https://github.com/LazyVim/LazyVim/assets/138600371/9fe04493-6af6-4051-b8ab-88ded5cd0a1d)

**how to use underline for cursor for insert mode and visual mode on lua language ?**

**Comment by Sam-programs** [comment]
see `:help vim.opt` and `:help 'guicursor'`
